packages/contracts/contracts/payoutStrategy/MerklePayoutStrategy/MerklePayoutStrategyImplementation.sol

```diff
@@ -32,7 +32,13 @@ contract MerklePayoutStrategyImplementation is IPayoutStrategy, Initializable {
   event DistributionUpdated(bytes32 merkleRoot, MetaPtr distributionMetaPtr);
 
   /// @notice Emitted when funds are distributed
-  event FundsDistributed(uint256 index, uint256 amount, address indexed token, address indexed sender, address indexed grantee);
+  event FundsDistributed(
+    uint256 amount,
+    address grantee,
+    address indexed token,
+    address indexed sender,
```

Comment:
As it's the same sender i.e the round operator, for all distributions, do we need to include it in all distribution events?
Also the sender is being emitted in the BatchPayoutSuccessful event.